udata_hydra/context.py

```diff
@@ -14,7 +14,7 @@
 }
 
 
-def monitor():
+def monitor() -> MagicMock:
```

Comment:
I think Monitor [died here](https://github.com/datagouv/hydra/pull/27) in silence. I had not realize it is only a Mock now ^^